astropy/units/astrophys.py

```diff
@@ -70,15 +70,16 @@
          doc="Electron mass",
          format={'latex': r'M_{e}', 'unicode': 'Mₑ'})
 # Unified atomic mass unit
-def_unit(['u', 'Da', 'Dalton'], 1.6605387e-27 * si.kg, namespace=_ns,
+def_unit(['u', 'Da', 'Dalton'], _si.u.value * si.kg, namespace=_ns,
          prefixes=True, exclude_prefixes=['a', 'da'],
          doc="Unified atomic mass unit")
 
 ##########################################################################
 # ENERGY
 
-def_unit(['Ry', 'rydberg'], 13.605692 * si.eV, namespace=_ns,
-         prefixes=True,
+def_unit(['Ry', 'rydberg'],
+         _si.Ryd.value * _si.c.value * _si.h.value / _si.e.value * si.eV,
+         namespace=_ns, prefixes=True,
```

Comment:
Nevertheless the value already given above is consistent with CODATA2010:
```
Rydberg constant times hc in eV                             13.605 692 53            0.000 000 30             eV
```
albeit slightly less accurate (should add the 53 at the end).  We should just define that as a constant in `astropy.constants` and use it here, rather than compute it ourselves.